src/util/ConfigManager/ConfigOption.cpp

```diff
@@ -35,19 +35,15 @@ std::string ConfigOption::availableTypesToString(const AvailableTypes& value) {
       return "string";
     } else if constexpr (std::is_same_v<T, int>) {
       return "integer";
+    } else if constexpr (std::is_same_v<T, size_t>) {
+      return "size_t";
     } else if constexpr (std::is_same_v<T, float>) {
       return "float";
-    } else if constexpr (std::is_same_v<T, std::vector<bool>>) {
-      return "list of booleans";
-    } else if constexpr (std::is_same_v<T, std::vector<std::string>>) {
-      return "list of strings";
-    } else if constexpr (std::is_same_v<T, std::vector<int>>) {
-      return "list of integers";
     } else {
       // It must be a vector.
       static_assert(ad_utility::isVector<T>);
       return absl::StrCat(
-          "list of ", availableTypesToString<typename T::value_type>(), "s");
+          "list of ", availableTypesToString<typename T::value_type>(), "'s");
```

Comment:
It was correct the way it was `ints` is correct, `int's` is not.

Reply:
Done, see [2a70fae](https://github.com/ad-freiburg/qlever/pull/977/commits/2a70faee0237700d3ddc946f668e4b45a656fe0a).